contrib/scrooge/src/python/pants/contrib/scrooge/tasks/scrooge_gen.py

```diff
@@ -135,20 +135,22 @@ def execute_codegen(self, target, target_workdir):
     self._must_have_sources(target)
 
     namespace_map = self._thrift_defaults.namespace_map(target)
+    compiler_args = self._thrift_defaults.compiler_args(target)
 
     partial_cmd = self.PartialCmd(
         language=self._validate_language(target),
         rpc_style=self._validate_rpc_style(target),
         namespace_map=tuple(sorted(namespace_map.items())) if namespace_map else (),
         default_java_namespace=self._thrift_defaults.default_java_namespace(target),
-        include_paths=target.include_paths)
+        include_paths=target.include_paths,
+        compiler_args=compiler_args)
```

Comment:
You'll never get here on a change to either the thrift defaults or the targets custom compiler args without updating the [fp strategy tied to all this](https://github.com/jsirois/pants/blob/master/contrib/scrooge/src/python/pants/contrib/scrooge/tasks/java_thrift_library_fingerprint_strategy.py#L29-L32). Test coverage that would reveal this miss as a break would be great to add as well.